src/pages/api/forum/events.json.ts

```diff
@@ -0,0 +1,30 @@
+import type { APIRoute } from 'astro';
+
+export const GET: APIRoute = async () => {
+  try {
+    const response = await fetch('https://forum.commercequest.space/api/v2/events', {
```

Comment:
we should directly filter by upcoming events and by previous events

Reply:
solved